mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -818,45 +818,60 @@ def __repr__(self):
 
 
 @PIPELINES.register_module()
-class IndoorPointSample(object):
-    """Indoor point sample.
+class PointSample(object):
+    """Point sample.
 
     Sampling data to a certain number.
 
     Args:
-        name (str): Name of the dataset.
         num_points (int): Number of points to be sampled.
+        sample_range (int, optional): The range where to sample points.
     """
 
-    def __init__(self, num_points):
+    def __init__(self, num_points, sample_range=None):
         self.num_points = num_points
-
-    def points_random_sampling(self,
-                               points,
-                               num_samples,
-                               replace=None,
-                               return_choices=False):
+        self.sample_range = sample_range
+
+    def _points_random_sampling(self,
+                                points,
+                                num_samples,
+                                sample_range=None,
+                                replace=None,
+                                return_choices=False):
         """Points random sampling.
 
         Sample points to a certain number.
 
         Args:
             points (np.ndarray | :obj:`BasePoints`): 3D Points.
             num_samples (int): Number of samples to be sampled.
-            replace (bool): Whether the sample is with or without replacement.
-            Defaults to None.
-            return_choices (bool): Whether return choice. Defaults to False.
-
+            sample_range (int, optional): Indicating the range where the points
+                will be sampled.
+                Defaults to None.
```

Comment:
why the type of sample_range is int?  if we use it as a threshold to get the far_inds and near_inds, may use the float is better?

Reply:
Sorry for that. It's reasonable to switch it to float.